_issues/bad_experience.md

```diff
@@ -6,18 +6,30 @@ categories: experience
   
 Q: Hillary has experience, but isn't it bad experience?  
   
-A: In the first Presidential debate, Donald Trump claimed that Hillary "has experience, but it's bad experience." This is utter nonsense. Hillary's record of public service spans four decades, starting with her first job out of law school. Here are just a few examples of [Hillary's many accomplishments](https://www.hillaryclinton.com/feed/seven-hillary-clintons-biggest-accomplishments/):
+A: In the first Presidential debate, Donald Trump claimed that Hillary "has experience, but it's bad experience." The words "bad experience" imply aa lack of success in her various roles. This is simply not true. Hillary's record of public service spans four decades, starting with her first job out of law school, and she has seen a [number of successes and accomplishments](https://www.hillaryclinton.com/feed/seven-hillary-clintons-biggest-accomplishments/) in that time:
```

Comment:
Do you think the items you have here are more of an answer to the question "why hasn't she accomplished anything in all her time in public office?"